data/core/get_started.md

```diff
@@ -0,0 +1,175 @@
+  - utter_ask_migration
+* switch{"current_api": "dialogflow"}
+  - utter_switch_dialogflow
+
+## story_002
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch{"current_api": "luis"}
+  - utter_switch_luis
+
+## story_003
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch
+  - utter_quickstart_nlu_only
+* thank
+  - utter_noworries
+
+
+## story_004
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+
+## story_005
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* switch
+  - utter_quickstart_nlu_only
+* thank
+  - utter_noworries
+* out_of_scope
+  - utter_docu
+
+## story_006
+* how_to_get_started
+  - utter_quickstart
+  - utter_ask_migration
+* out_of_scope
+  - utter_docu
+
+## Generated Story -5718603718660805764
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+    - utter_quickstart
+    - utter_ask_migration
+* switch{"current_api": "luis"}
+    - utter_switch_luis
+* thank
+    - utter_noworries
+
+## Generated Story 4922905359326269181
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+    - utter_quickstart
+    - utter_ask_migration
+* thank
+    - utter_noworries
+
+
+
+## Generated Story 6866562783165718158
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* switch{"current_api": "dialogflow"}
+    - utter_switch_dialogflow
+
+
+## tech question
+* technical_question
+  - utter_cant_answer_tech_question
+  - utter_docu
+
+## tech question again
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* switch{"current_api": "dialogflow"}
+    - utter_switch_dialogflow
+* technical_question
+  - utter_cant_answer_tech_question
+  - utter_docu
+
+
+## tech question again
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* technical_question
+  - utter_cant_answer_tech_question
+  - utter_docu
+
+
+## Generated Story -2693866830817074294
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started
+    - utter_quickstart
+    - utter_ask_migration
+* switch{"current_api": "dialogflow"}
+    - utter_switch_dialogflow
+
+
+## Generated Story 4741357980983644475
+* greet
+    - utter_greet
+    - utter_inform_privacypolicy
+* mood_confirm
+    - utter_awesome
+    - utter_ask_goal
+* how_to_get_started{"product": "core"}
```

Comment:
we need a story for "product":"nlu" too

Reply:
Also a stories where the user says `* how_to_get_started` and then afterwards says he wants to know more about core or NLU.